### ISSUE TITLE
Regarding the Talk

### ISSUE THREAD

**Issue by Praneet460** [wwgdelhi/Talks#4]
Hello, I am interested in giving a  talk. 
But I want to know can I choose any of the topics for the talk like machine learning, data science or Python.

**Comment by RN0311** [comment]
No, you're free to choose from any topic which you want. And there is no restriction on any specific language.

**Comment by Praneet460** [comment]
I would be happy to give a talk on Introduction To Machine Learning Using Python
The code and the slides can be found on [Here](https://github.com/Praneet460/Introduction-To-ML-Workshop)
